src/content/plugins/define-plugin.md

```diff
@@ -68,8 +68,7 @@ After passing through webpack with no minification results in:
 if (!true) {
   console.log('Debug info');
 }
-if (true) {
-  console.log('Production log');
+if (fasle) {
```

Comment:
Shouldn't be `false`?

Reply:
the compiled file' content is `if(false){}`.  there's no content inside the curly brace